mappings.py

```diff
@@ -0,0 +1,139 @@
+    "nvram_invmatch": "libinject_nvram_invmatch",
+    "nvram_parse_nvram_from_file": "libinject_parse_nvram_from_file",
+}
+
+// Alternative names for the same function
+base_aliases = {
+    "nvram_load": "libinject_nvram_init",
+    "nvram_loaddefault": "libinject_ret_true",
+    "_nvram_get": "libinject_nvram_get",
+    "nvram_get_state": "libinject_nvram_get_int",
+    "nvram_set_state": "libinject_nvram_set_int",
+    "nvram_restore_default": "libinject_nvram_reset",
+    "nvram_upgrade": "libinject_nvram_commit",
+    "get_default_mac": "libinject_ret_true",
+    "VCTGetPortAutoNegSetting": "libinject_ret_false1",
+    "agApi_fwGetFirstTriggerConf": "libinject_ret_true1",
+    "agApi_fwGetNextTriggerConf": "libinject_ret_true1",
+    "artblock_get": "libinject_nvram_get",
+    "artblock_fast_get": "libinject_nvram_safe_get",
+    "artblock_safe_get": "libinject_nvram_safe_get",
+    "artblock_set": "libinject_nvram_set",
+    "nvram_flag_set": "libinject_ret_true",
+    "nvram_flag_reset": "libinject_ret_true",
+    "nvram_master_init": "libinject_ret_false",
+    "nvram_slave_init": "libinject_ret_false",
+    "apmib_init": "libinject_ret_true",
+    "apmib_reinit": "libinject_ret_true",
+    "apmib_update": "libinject_ret_true",
+    "WAN_ith_CONFIG_SET_AS_STR": "libinject_nvram_nset",
+    "WAN_ith_CONFIG_SET_AS_INT": "libinject_nvram_nset_int",
+    "acos_nvram_init": "libinject_nvram_init",
+    "acos_nvram_get": "libinject_nvram_get",
+    "acos_nvram_read": "libinject_nvram_get_buf",
+    "acos_nvram_set": "libinject_nvram_set",
+    "acos_nvram_loaddefault": "libinject_ret_true",
+    "acos_nvram_unset": "libinject_nvram_unset",
+    "acos_nvram_commit": "libinject_nvram_commit",
+    "acosNvramConfig_init": "libinject_nvram_init",
+    "acosNvramConfig_get": "libinject_nvram_get",
+    "acosNvramConfig_read": "libinject_nvram_get_buf",
+    "acosNvramConfig_set": "libinject_nvram_set",
+    "acosNvramConfig_write": "libinject_nvram_set",
+    "acosNvramConfig_unset": "libinject_nvram_unset",
+    "acosNvramConfig_match": "libinject_nvram_match",
+    "acosNvramConfig_invmatch": "libinject_nvram_invmatch",
+    "acosNvramConfig_save": "libinject_nvram_commit",
+    "acosNvramConfig_save_config": "libinject_nvram_commit",
+    "acosNvramConfig_loadFactoryDefault": "libinject_nvram_loaddefault",
+    "nvram_commit_adv": "libinject_nvram_commit",
+    "nvram_unlock_adv": "libinject_ret_true",
+    "nvram_lock_adv": "libinject_ret_true",
+    "nvram_check": "libinject_ret_true",
+    "envram_get_func": "libinject_envram_get",
+    "nvram_getf": "libinject_envram_getf",
+    "envram_set_func": "libinject_envram_set",
+    "nvram_setf": "libinject_envram_setf",
+    "envram_unset_func": "libinject_envram_unset",
+}
+
+// All variables together
+default_lib_aliases = {
+    string_introspection,
+    atheros_broadcom,
+    realtek,
+    netgear_acos,
+    zyxel_or_edimax,
+    ralink,
+    base_names,
+    base_aliaes
+}
```

Comment:
Nitpick: file missing trailing newline

Reply:
I think I added one? The github web ui isn't super clear about it though.